frontend/src/index.jade

```diff
@@ -155,14 +179,29 @@ html
                       right: (getSlicePos(j, user.commits.length) * 100) + '%'\
                     }"
                   )
-              .summary-chart__contrib(
-                v-bind:title=
-                    "'Total contribution from ' + minDate + ' to ' + maxDate + ': ' + user.totalCommits + ' lines'"
-               )
-                .summary-chart__contrib--bar(
-                  v-for="width in getContributionBars(user.totalCommits)",
-                  v-bind:style="{ width: width+'%' }"
-                )
+              .summary-chart__contribution
+                template(v-if="filterBreakdown")
+                  .summary-chart__contrib(
+                      v-for="(widths, fileFormat) in getFileFormatContributionBars(user.fileFormatContribution)"
+                  )
+                    .summary-chart__contrib--bar--fileformat(
+                        v-for="width in widths",
+                        v-bind:style="{ width: width + '%',\
+                            'background-color': contributionBarColors[fileFormat] }",
+                        v-bind:title="fileFormat + ': ' + user.fileFormatContribution[fileFormat] + ' lines, '\
+                            + 'total: ' + user.totalCommits + ' lines ' + '(contribution from ' + minDate + ' to '\
+                            + maxDate + ')'"
+                    )
+                template(v-else)
+                  .summary-chart__contrib(
+                      v-bind:title=
+                          "'Total contribution from ' + minDate + ' to ' + maxDate + ': ' + user.totalCommits\
```

Comment:
should follow the above style, such that the value of the title does not begin from the next line.

E.g. 
```
v-bind:title="'Total contribution from ' + minDate + ' to ' + maxDate + ': '\
    + user.totalCommits + ' lines'"
```